k8s/runners/staging/release.yaml

```diff
@@ -46,7 +46,8 @@ data:
           environment = ["FF_GITLAB_REGISTRY_HELPER_IMAGE=1"]
           [runners.kubernetes]
             privileged = false
-            cpu_request = "9000m"
+            cpu_request = "750m"
+            memory_request = "2Gi"
```

Comment:
@kotfic could you add your changes to this runner?  This is the runner we use in staging.